frontend-project/src/components/Table/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { PageHeaderWrapper } from '@ant-design/pro-layout';
+import ProTable, { ActionType } from '@ant-design/pro-table';
+import React, { FC, useRef } from 'react';
+import { formatMessage } from 'umi-plugin-react/locale';
+
+const Table: FC<{ type: string; columns; fetchData }> = props => {
+  const actionRef = useRef<ActionType>();
+
+  const showTotal = (total, range) =>
```

Comment:
Zdaję sobie sprawę, że ostatnio dałem approve PR, który również nie definiował typów podobnej metody ale było to przeoczenie i tutaj bym Cię prosił o zdefiniowanie typów dla total i range

Poza tym coś kojarzę podobną metodę w innym pliku. Może w ramach DRY warto ją przenieść do jakichś utilsów jeśli to możliwe